torch/nn/quantized/modules/conv.py

```diff
@@ -38,14 +39,31 @@ def __init__(self, weight, bias, scale, zero_point, dtype,
         self.groups = groups
         self.padding_mode = padding_mode
 
-        self.weight = weight
-        self._packed_weight = ops.quantized.fbgemm_conv_prepack(weight, groups)
-        self.bias = bias  # torch.nn.Parameter(bias, requires_grad=False)
+        self._weight = weight  # Store the original weight for future reference.
```

Comment:
Please remove _weight. Quantized modules should be smaller in size and we should not store multiple copies of weights. Let us generate the unpacked weights when it is needed.